packages/conda-store/src/condaStore.ts

```diff
@@ -35,6 +35,15 @@ interface IPaginatedResult<T> {
   status?: string;
 }
 
+// See class CondaSpecification in conda_store_server/schema.py.
+// https://github.com/Quansight/conda-store/blob/main/conda-store-server/conda_store_server/schema.py#L111
+interface ICondaStoreSpecification {
```

Comment:
Not technically necessary (i.e., TypeScript won't complain) but I thought it might be good to add this type definition